test/simple_recipes.hpp

```diff
@@ -50,6 +50,10 @@ class simple_recipe_base: public recipe {
         return catalogue_;
     }
 
+    void add_ion(const char* name, int charge, double iconc, double econc) {
+        cell_gprop_.ion_default[name] = {charge, iconc, econc};
```

Comment:
Here add an ions species and set its charge and concentrations. How do we set the default reversal potential?

Reply:
While this is just a helper for the simple recipe class, in the current version of the cable cell global properties, there is not a mechanism for providing this in the ion table. This facility though is part of the big pending reversal potential PR.